src/main/java/io/vertx/core/http/impl/HttpServerImpl.java

```diff
@@ -304,6 +340,13 @@ protected void configurePipeline(ChannelHandlerContext ctx, String protocol) thr
     return this;
   }
 
+  // Visible for testing
+  protected void handleConnectionException(ChannelHandlerContext ctx, Throwable cause) {
```

Comment:
ok so more like:

1. the close is done in Netty's handler
2. not a method to override but instead an `Handler<Throwable>` called `connectionExceptionHandler` that by default logs the cause and can be `set` in the test